### PR TITLE
Hangman

### PR DESCRIPTION
**Pull request by AHShahabi**

### CODE REVIEW

File: Hangman.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+def get_valid_word(words):
+    word = random.choice(words) # Randomly chooses sth from words list.
+    while "-" in word or " " in word:
+        word = random.choice(words)
+    
+    return word.upper()
+
+player_name = input ("Welcome! what's your name? ")
+print (f"Hello {player_name}. I hope enjoy the game! if you won less than 45 second, you are a hero without a cape {chr(171)} {chr(164)} {chr(187)}")
+time.sleep (3)
```
Comment (AlirezaEbrahimkhani) [inline Hangman.py]: Why do we need sleep here?

File: Hangman.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+def Hangman():
+    word = get_valid_word (words)
```
Comment (AlirezaEbrahimkhani) [inline Hangman.py]: Remove space before (words)

Comment (AHShahabi) [reply Hangman.py]: done

File: Hangman.py (hunk 1 above)
Comment (AHShahabi) [reply Hangman.py]: To draw the user's attention to the welcome sentence

**Pull request merged**